baselines/statavg/statavg/dataset_preparation.py

```diff
@@ -0,0 +1,75 @@
+"""Prepare the dataset."""
+
+import os
+from typing import List, Tuple
+
+import pandas as pd
+from omegaconf import DictConfig
+from pandas import DataFrame
+from sklearn.model_selection import train_test_split
+
+
+def get_split_dataset(
+    path_to_dataset: str, include_testset: DictConfig
+) -> Tuple[DataFrame, DataFrame]:
+    """Load and split datsaet to train (client) and test (server)."""
+    script_dir = os.path.dirname(os.path.abspath(__file__))
+    dataset_path = os.path.join(script_dir, path_to_dataset, "dataset.csv")
+    dataset = pd.read_csv(dataset_path)
+
+    # remove NaN
+    dataset = dataset.dropna(axis=0, how="any")
+
+    # keep label
+    Y = dataset[["type"]]
+
+    # remove irrelevant features
+    dataset = dataset.drop(["ts", "PID", "CMD", "label"], axis=1)

```

Comment:
The code crashes here because there is no `ts` column. Columns present when downloading the dataset are:

```csv
PID,MINFLT,MAJFLT,VSTEXT,VSIZE,RSIZE,VGROW,RGROW,MEM,CMD,label,type
```

Reply:
Thanks @jafermarq, you are correct. ```ts``` which stands for timestamp, was removed from the feature space in the new version of the dataset. I will make the corresponding change as you indicated.